--- origsrc/awn-extras-applets-0.2.6/src/Makefile.am	2008-02-18 18:58:04.000000000 -0600
+++ src/awn-extras-applets-0.2.6/src/Makefile.am	2008-10-30 12:40:30.820786100 -0500
@@ -8,7 +8,6 @@
 	cairo-menu		\
 	cairo-menu-classic	\
 	calendar		\
-	filebrowser		\
 	main-menu		\
 	plugger			\
 	stacks			\
@@ -37,14 +36,12 @@
 if WITH_GNOME
 GCONF_APPLETS +=	\
 	MiMenu		\
-	pandora			\
 	$(NULL)
 endif
 
 SUBDIRS =			\
 	libawn-extras		\
 	affinity		\
-	battery-applet		\
 	mail			\
 	notification-area	\
 	places			\
@@ -54,7 +51,6 @@
 	shiny-switcher		\
 	showdesktop		\
 	tsclient-applet		\
-	volume-control		\
 	wobblyzini		\
 	$(GCONF_APPLETS)	\
 	$(VALA_APPLETS)		\
