--- origsrc/awn-extras-applets-0.3.2.1/src/libawn-extras/Makefile.am	2009-02-19 15:04:37.000000000 -0600
+++ src/awn-extras-applets-0.3.2.1/src/libawn-extras/Makefile.am	2009-03-10 23:08:48.524825800 -0500
@@ -28,7 +28,7 @@ libawn_extras_la_SOURCES = \
 	$(source_c)
 	
 libawn_extras_la_LIBADD = $(AWN_LIBS) $(LIBNOTIFY_LIBS)
-libawn_extras_la_LDFLAGS = -version-info 0:1:0
+libawn_extras_la_LDFLAGS = -version-info 0:1:0 -no-undefined
 
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = awn-extras.pc
--- origsrc/awn-extras-applets-0.3.2.1/src/libawn-extras/bindings/python/Makefile.am	2009-02-19 15:04:28.000000000 -0600
+++ src/awn-extras-applets-0.3.2.1/src/libawn-extras/bindings/python/Makefile.am	2009-03-10 23:16:53.352950800 -0500
@@ -22,7 +22,7 @@ INCLUDES = \
 pyawn_extrasexecdir = $(PYTHON_SITE_PKG)/awn/extras
 pyawn_extrasexec_LTLIBRARIES = _extras.la
 
-common_ldflags = -module -avoid-version
+common_ldflags = -module -avoid-version -no-undefined
 
 AWN_EXTRAS_DEFS = awn_extras.defs
 AWN_EXTRAS_OVERRIDE = awn_extras.override
